examples/classification/main.py

```diff
@@ -76,6 +88,16 @@ def main(argv):
     if config.dist_url == "env://":
         config.update_from_env()
 
+    mlflow.set_tracking_uri(osp.join(config.log_dir, 'mlruns'))
```

Comment:
mlflow_runs or mlflow_experiments

Reply:
For now there are some problems with renaming main mlflow tracking directory